src/sdc11073/provider/sco.py

```diff
@@ -311,13 +306,15 @@ def unregister_operation_by_handle(self, operation_handle: str):
     def get_operation_by_handle(self, operation_handle: str) -> OperationDefinition:
         return self._registered_operations.get(operation_handle)
 
-    def enqueue_operation(self, operation: OperationDefinition, request, operation_request):
-        """ enqueues operation "operation".
+    def enqueue_operation(self, operation: OperationDefinition, request, operation_request, transaction_id: int):
+        """Enqueue operation "operation".
+
         :param operation: a callable with signature operation(request, mdib)
         :param request: the soapEnvelope of the request
-        @return: a transaction Id
+        :param operation_request: the argument for the operation
+        :param transaction_id:
```

Comment:
missing doc

Reply:
the whole provider sub-package will be updated soon with typing etc.